README.md

```diff
@@ -1,58 +1,53 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
-# Project Name
-
-> Description the project.
-
+# Wemdigo
+- A ruby on rails app for keep track of your expenses. Users can create categories and transactions to which.
+- Notice: This is a mobile app so open it with your phone or make sure you are using a 400px width screen or resize the view using browser's inspection.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Ruby
+- RoR
 
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
+## Live Demo
+- [Click Me!](https://desolate-stream-21599.herokuapp.com/)
 
+- PS : Heroku is not the best to host a RoR app, it's better that you'd clone the project and follow
+the instructions below for better experience.
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
+- Run git clone git@github.com:AmineHLub/Wemdigo.git to get a local clone
 
 ### Prerequisites
 
-### Setup
+- Make sure you have Ruby and RoR installed
+- Run bundle install
 
-### Install
+### Run
 
-### Usage
+Run rails db:create db:migrate
 
-### Run tests
+P.S : if you are coming from a previous version, you might want to delete the database by running rails db:drop (BECAREFUL that all previous data will be deleted)
 
-### Deployment
+Run rails s
 
+To view generated mails, check terminal logs
 
+### Run tests
 
-## Authors
+Run rspec spec/integration for integration tests
 
-👤 **Author1**
+👤 **Mohamed Amine**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@AmineHLub](https://github.com/AmineHLub)
+- Twitter: [@Amino47612441](https://twitter.com/Amino47612441)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mohamed-amine-hajltaief-b18863163/)
 
-👤 **Author2**
+## 🤝 Contributing
+- Microverse
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- [Gregoire Vella on Behance](https://www.behance.net/gregoirevella)
 
-## 🤝 Contributing
+  - [Her design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=)
```

Comment:
- [ ] The contributing section of the README should be a guide on how other developers can contribute to a project. It answers the question, how can one collaborate with you on this project?

  The details you've included should actually be in the `Acknowledgements` section. Please move them to the Acknowledgement section.
  
  Then, the details you have in the `Acknowledgements` section are just the general guidelines on how one should write the acknowledgments. Please replace them with the details you have here.
  
  Lastly, remember to re-write this `Contributing` section.

Reply:
This change has been done well